src/client/app.constants.js

```diff
@@ -0,0 +1,4 @@
+export const COOKIES = {
+  ACCESS_TOKEN: 'access_token',
+  EXPIRED_AT: 'expired_at',
```

Comment:
Do we need expire_at at all?

Related comment here: https://github.com/paralect/koa-api-starter/pull/42/files